adapters/33across/33across.go

```diff
@@ -104,6 +119,19 @@ func preprocess(request *openrtb.BidRequest) error {
 	siteCopy.ID = ttxExt.SiteId
 	request.Site = &siteCopy
 
+	// Validate Video if it exists
+	if imp.Video != nil {
+		videoCopy, err := validateVideoParams(imp.Video, impExt.Ttx.Prod)
+
+		imp.Video = videoCopy
+
+		if err != nil {
```

Comment:
Is it desired to set the `imp.Video` to `nil` if there's an error?

Reply:
Yes, so long as there is at least one valid format `banner` or `video` pass that to downstream. Only fail if both formats error out during MakeRequests